rules/windows/process_creation/proc_creation_win_redirect_local_admin_share.yml

```diff
@@ -0,0 +1,26 @@
+title: Suspicious Redirect To Local Admin Share
+id: 77d570aa-4e72-4949-98ff-24cdeec16787
+status: experimental
+description: Detects output redirection to the local admin share (ADMIN$) via the commandline
+author: Nasreddine Bencherchali
+references:
+    - http://blog.talosintelligence.com/2022/09/lazarus-three-rats.html
+date: 2022/09/09
+tags:
+    - attack.execution
+    - attack.lateral_movement
+    - attack.t1021.001
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains:
```

Comment:
what do you think of 
````yaml
CommandLine|contains\all:
    - '>'
    - '\\\\127.0.0.1\\ADMIN$\\'
```` 
To keep the logic use double \ too

Reply:
Just saw that there is already ab9e3b40-0c85-4ba1-aede-455d226fd124 I'll delete mine and update the other